runtimes/pythonrt/runner/syscalls.py

```diff
@@ -123,6 +123,38 @@ def ak_next_event(self, args, kw):
 
         return AttrDict(data) if isinstance(data, dict) else data
 
+    def ak_join(self, args, kw):
```

Comment:
This is old code, the new syscalls have regular Python signature.
```suggestion
    def ak_join(self,signal, *signals):
```